csharp/Ice/async/Client.cs

```diff
@@ -15,121 +14,107 @@ public static int Main(string[] args)
 
         try
         {
-            //
-            // using statement - communicator is automatically destroyed
-            // at the end of this statement
-            //
-            using(var communicator = Ice.Util.initialize(ref args, "config.client"))
+            using (Ice.Communicator communicator = Ice.Util.initialize(ref args, "config.client"))
```

Comment:
Why get rid of the comment?

Reply:
What `using` is about is basic C#. It's odd to have this comment in every single server.